test/models/admin_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
The `test` folder should not even be here. Can you delete it?

(It's not in `master` branch, that's weird...)